### DISCUSSION TITLE
Changing font

### DISCUSSION THREAD

**Discussion by UsernameHackerManisnotavailable** [phcode-dev/phoenix#1929] [Q&A] [answered]
So I was looking through the view < themes tab and I found a little box at the bottom with some text in it that is the font. I am wondering if that font is for the UI and how would I get a font into that format that I could change it.
![Screenshot 2024-11-15 11 16 28 AM](https://github.com/user-attachments/assets/c046adf9-b240-46a6-a320-ad70a8233d2b)

**Comment by devvaannsh** [comment, marked as answer]
Hi! The font setting is for the editor, not the UI.

To change the font :- first of all, make sure the font is installed on your computer. Then, in the Font Family input box, just type the font name exactly as it appears (case-sensitive). No need to add font-family:—just list the font names directly.

For example, type it like this: 'Fira Code', 'Hack', 'Term'.

Hope this helps!

**Reply by UsernameHackerManisnotavailable** [reply]
Thanks!
